robotframework_reportportal/service.py

```diff
@@ -214,7 +218,7 @@ def start_keyword(self, keyword, ts=None):
         """
         start_rq = {
             'description': keyword.doc,
-            'has_stats': False if keyword.parent_type == 'KEYWORD' else True,
+            'has_stats': False,
```

Comment:
This was standard behavior of Robot Listener prior to refactoring. We can't change it that simple, because our users expect different.
I'm going to implement another story which will introduce this switch as a variable, but default behavior will stay as is.

Reply:
This is a correct behavior since this logic completely conforms the RP [nested item structure](https://github.com/reportportal/reportportal/issues/275). Otherwise, you will get incorrect results similar to [this one](https://github.com/reportportal/agent-Python-RobotFramework/issues/43).

Reply:
Then we should fix it in different way, since a regression was introduced by this fix.

How it looked previously.
Suite level:
![Screen Shot 2021-04-14 at 11 17 35 AM](https://user-images.githubusercontent.com/4364466/114677487-080de580-9d13-11eb-8a1a-79e1396111c3.png)
Test level:
![Screen Shot 2021-04-14 at 11 18 08 AM](https://user-images.githubusercontent.com/4364466/114677564-1a881f00-9d13-11eb-8f4a-adf5401a08f6.png)
Step level:
![Screen Shot 2021-04-14 at 11 18 35 AM](https://user-images.githubusercontent.com/4364466/114677632-2b389500-9d13-11eb-86c6-43df75ae0e4f.png)
Hot it looks now.
Suite level:
![Screen Shot 2021-04-14 at 11 19 16 AM](https://user-images.githubusercontent.com/4364466/114677741-460b0980-9d13-11eb-82f8-97f785c473f5.png)
Test level:
![Screen Shot 2021-04-14 at 11 19 53 AM](https://user-images.githubusercontent.com/4364466/114677843-5a4f0680-9d13-11eb-896d-5881d0504bc9.png)
Step level:
![Screen Shot 2021-04-14 at 11 20 16 AM](https://user-images.githubusercontent.com/4364466/114677905-676bf580-9d13-11eb-8c7f-5900e3f12db0.png)

So it breaks statistics (7 items instead of 1) and step display and completely unexpected for users.

I can think over different solution if you want, but this one inappropriate.

Reply:
I'd like to see the robot file you used in your testing before making a conclusion on what structure is correct.

Reply:
@iivanou sure, here: https://github.com/reportportal/examples-python/blob/master/robotframework/robot/logging.robot

Reply:
With the initial code base:
![image](https://user-images.githubusercontent.com/8349078/114881184-70d88900-9e0b-11eb-8508-4114f360f51b.png)
![image](https://user-images.githubusercontent.com/8349078/114881300-8c439400-9e0b-11eb-9e60-115fcc0d7ce7.png)
Everything's correct.

Reply:
@iivanou initial agent version does not work with RF 4, try 3.2.2

Reply:
Why? It works...